packages/manager/src/features/Images/ImagesLanding/ImageRegions/ManageImageRegionsForm.tsx

```diff
@@ -113,6 +114,28 @@ export const ManageImageReplicasForm = (props: Props) => {
         </Link>{' '}
         for details on managing your Linux system's disk space.
       </Typography>
+      <DismissibleBanner
+        preferenceKey={`manage-replicas-${image?.id}`}
```

Comment:
Not sure what the exact requirement was here, but I feel like we should make this not dismissible. 

We also may want to create ourselves a "todo" ticket to remove this banner once billing for images has started

Reply:
Or at the very least, don't make the banner unique on a per-image basis 

```suggestion
        preferenceKey={`manage-replicas-pricing-notice`}
```

Reply:
Sure, converted to not dismissible. Got the UX approval as well.